src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -228,24 +225,42 @@ const WalletTable = ({
     walletCardData,
   } = useWalletTable({ filters, t, walletData })
 
+  // Context API
+  const { supportedLanguage } = useContext(WalletSupportedLanguageContext)
+
   return (
     <Container>
       <WalletContentHeader>
         <Th sx={{ textAlign: "start !important" }}>
-          {filteredWallets.length === walletCardData.length ? (
-            <Text ps={{ base: 2, md: 0 }} as="span">
-              {t("page-find-wallet-showing-all-wallets")} (
-              <strong>{walletCardData.length}</strong>)
-            </Text>
-          ) : (
-            <Text ps={{ base: 2, md: 0 }} as="span">
-              {t("page-find-wallet-showing")}{" "}
-              <strong>
-                {filteredWallets.length} / {walletCardData.length}
-              </strong>{" "}
-              {t("page-find-wallet-wallets")}
+          <Flex justifyContent="space-between" px={{ base: 2.5, md: 0 }}>
+            <Text
+              display={{ base: "block", lg: "none" }}
```

Comment:
Not a requirement, just noting we can use this syntax as well:
```suggestion
              hideFrom="lg"
```

Reply:
Thanks!, fixed in `find-wallet-revamp` branch